### PR TITLE
Amend kubeadm join doc for node preparation

### PR DESCRIPTION
**Pull request by tengqm**
We need to clarify that worker nodes need to be prepared in nearly the same way as the control plane nodes.

### CODE REVIEW

**Review by celestehorgan — COMMENTED** [review]
/lgtm

/sig node

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -292,11 +292,13 @@ The nodes are where your workloads (containers and Pods, etc) run. To add new no
 
 * SSH to the machine
 * Become root (e.g. `sudo su -`)
+* [Install runtime](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime)
```
Comment (jimangel) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:295]: minor nit
```suggestion
* [Install a runtime](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime)
```

**Comment by reylejano** [comment]
Preview of edited page:
https://deploy-preview-29127--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -292,11 +300,13 @@ The nodes are where your workloads (containers and Pods, etc) run. To add new no
 
 * SSH to the machine
 * Become root (e.g. `sudo su -`)
+* [Installing a runtime](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime)
```
Comment (reylejano) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:303]: Small nit to keep parallelism with the verb tense in the list
```suggestion
* [Install a runtime](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime)
```

**Comment by reylejano** [comment]
re-applying lgtm from sig docs and sig cluster-lifecycle
/lgtm

**Pull request merged**